src/sentry/tasks/post_process.py

```diff
@@ -203,6 +205,19 @@ def post_process_group(
                 ):
                     safe_execute(callback, event, futures)
 
+            has_commit_key = "workflow-owners-ingestion:org-{}-has-commits".format(
+                event.project.organization_id
+            )
+            org_has_commit = cache.get(has_commit_key)
+            if org_has_commit is None:
+                org_has_commit = Commit.objects.filter(
+                    organization_id=event.project.organization_id
+                ).exists()
+                cache.set(has_commit_key, org_has_commit, 3600)
+
+            if org_has_commit and features.has("projects:workflow-owners-ingestion", event.project):
+                process_suspect_commits(event=event)
```

Comment:
Just to be safe, let's wrap this in a try/catch and just log any exceptions rather than failing.

Reply:
Done and done!